java/code/src/com/redhat/rhn/frontend/dto/SetupWizardProductDto.java

```diff
@@ -215,6 +215,12 @@ public boolean isFinished() {
         }
     };
 
+    /** The product id */
+    private Long id;
+
+    /** The database id */
+    private Long databaseId;
```

Comment:
I would rename these as this could become confusing with how we named these at other placed.

The id column in the database which normally has also a sequence is always called just `id`.
The ID of the product as it comes from SCC is called `product_id`.

This seems to turn it around and could become a problem.
Please call the databaseID => id and the id => productId .

Reply:
I agree with you. I named it like this because in the class `MgrSyncProductDto` we have already a field named `id`, which is actually the `reportId`. I will refactor both classes to stay consistent.

Reply:
I [refactored all the affected DTOs](https://github.com/uyuni-project/uyuni/pull/4671/commits/eb7253895afe780cb4760c1483d5f12fe71c683b). Now the naming is consistent with the database.